src/frontend/icons/Search.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import { cn } from '../utils';
+const Search: FC<{ className?: string }> = ({ className }) => {
```

Comment:
you can use the svg directly no need to transform it into component since you dont use the props to change values in 'path-d' or 'viewbox' etc...